website/site/content/download/personal-edition/ubuntu.md

```diff
@@ -147,6 +147,44 @@ Change the dbconfig setting to use the postgres database you created:
 "dbconfig": "postgres://boardsuser:boardsuser-password@localhost/boards?sslmode=disable&connect_timeout=10",
 ```
 
+## Install MySQL 
+
+Alternate to Postgress you also can store your data in a MySQL/MariaDB database.
+16). To install, run:
+
+```
+sudo apt install mariadb-server mariadb-client
+```
+
+Log in as root in your database:
+```
+sudo mysql 
+```
+
+On the MySQL prompt, run the following commands (**change the user/password** to your own values):
+<pre>
+CREATE DATABASE boards;
+GRANT ALL on boards.* to <b>'boardsuser'@'localhost'</b> identified by '<b>boardsuser-password</b>';
+
+</pre>
+
+Exit the mysql-prompt:
+```
+exit
+```
+
+Edit the Focalboard config.json:
+
+```
+nano /opt/focalboard/config.json
+```
+
+Change the dbconfig setting to use the MySQL database you created:
+```
+"dbtype": "mysql",
+"dbconfig": "mysql://boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
```

Comment:
Accoding to my test & #301 this shoud be:
```
"dbconfig": "boardsuser:boardsuser-password@tcp(127.0.0.1:3306)/boards",
```

Reply:
Indeed! Thanks for noticing